packages/web-react/src/components/FieldGroup/FieldGroup.stories.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { ValidationStates } from '../../constants';
+import { Radio } from '../Radio';
+import { Checkbox } from '../Checkbox';
+import FieldGroup from './FieldGroup';
+import ReadMe from './README.md';
+
+const meta: Meta<typeof FieldGroup> = {
+  title: 'Components/FieldGroup',
+  component: FieldGroup,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'select',
+      options: ['boxes', 'radios', 'checkboxes'],
+      description:
+        'This is the place for the content of the FieldGroup. In the real code you can pass in any ' +
```

Comment:
I would rather use string literals than concatenating strings using the `+` sign. Is it possible to switch without breaking the description?